### PR TITLE
enhance(schedule): add branch field to form

### PR DESCRIPTION
**Pull request by claire1618**
Adding a branch field to scheduled builds, so this adds a column in the schedules builds table and an input box for the user to specify the branch wanted for a scheduled build

**Before:**
Screenshot 2023-08-29 at 2 10 43 PM
Screenshot 2023-08-29 at 2 10 37 PM

**After:**
Screenshot 2023-08-30 at 8 37 51 AM
Screenshot 2023-08-29 at 2 06 52 PM

### CODE REVIEW

**Comment by JordanSussman** [comment]
Is the branch going to be a required field that the user provides? If not, can you update the input box to clarify the behavior when no branch is provided?

File: src/elm/Pages/Schedules/Form.elm [hunk 1]
```diff
@@ -168,6 +170,31 @@ viewEnabledCheckbox enableUpdate =
...
+{-| viewBranchNameInput : renders branch input box
+-}
+viewBranchNameInput : String -> Bool -> Html Msg
+viewBranchNameInput val2 disable =
+    section [ class "form-control", class "-stack" ]
+        [ label [ class "form-label", for <| "schedule-branch-name" ] 
+                [ strong [] [ text "Branch" ]
+                , span
+                    [ class "field-description" ]
+                    [ em [] [ text "(Leave blank to default to main)" ]
```
Comment (ecrupper) [inline src/elm/Pages/Schedules/Form.elm:182]:
```suggestion
                    [ em [] [ text "(Leave blank to use default branch)" ]
```

Won't always be main

File: docker-compose.yml [hunk 2]
```diff
@@ -42,7 +42,6 @@ services:
       DATABASE_ENCRYPTION_KEY: 'C639A572E14D5075C526FDDD43E4ECF6'
       QUEUE_DRIVER: redis
       QUEUE_ADDR: 'redis://redis:6379'
-      QUEUE_PRIVATE_KEY: 'tCIevHOBq6DdN5SSBtteXUusjjd0fOqzk2eyi0DMq04NewmShNKQeUbbp3vkvIckb4pCxc+vxUo+mYf/vzOaSg=='
```
Comment (ecrupper) [inline docker-compose.yml:45]: We'll want to keep these compose values

**Review by ecrupper — CHANGES_REQUESTED** [review]
I think some Cypress tests need addressed too, but other than that there were a couple things I noticed

File: cypress/fixtures/schedule.json [hunk 3]
```diff
@@ -8,6 +8,7 @@
   "updated_at": 1685037152,
   "updated_by": "CookieCat",
   "scheduled_at": 0,
+  "branch": "master",
```
Comment (plyr4) [inline cypress/fixtures/schedule.json:11]:
```suggestion
  "branch": "main",
```

Comment (plyr4) [inline cypress/fixtures/schedule.json:11]: not a comment for you but we should really change all these master references to main

File: src/elm/Pages/Schedules/Form.elm [hunk 4]
```diff
@@ -168,6 +170,31 @@ viewEnabledCheckbox enableUpdate =
...
+            [ disabled disable
+            , value val2
+            , onInput <|
+                OnChangeStringField "branch"
+            , placeholder "Branch Name"
+            , id "schedule-branch-name"
+            , Util.testAttribute "schedule-branch-name"
+            ]
+            []
+        ]
```
Comment (plyr4) [inline src/elm/Pages/Schedules/Form.elm:195]:
```suggestion
viewBranchNameInput : String -> Bool -> Html Msg
viewBranchNameInput val disable =
    section [ class "form-control", class "-stack" ]
        [ label [ class "form-label", for <| "schedule-branch-name" ]
            [ strong [] [ text "Branch" ]
            , span
                [ class "field-description" ]
                [ em [] [ text "(Leave blank to use default branch)" ]
                ]
            ]
        , input
            [ disabled disable
            , value val
            , onInput <|
                OnChangeStringField "branch"
            , placeholder "Branch Name"
            , id "schedule-branch-name"
            , Util.testAttribute "schedule-branch-name"
            ]
            []
        ]
```

Comment (plyr4) [reply src/elm/Pages/Schedules/Form.elm:195]: very tiny nitpick for consistency

Comment (claire1618) [reply src/elm/Pages/Schedules/Form.elm]: Fixed!

**Pull request merged**